src/main/java/com/codecool/virtualpub/logic/Pub.java

```diff
@@ -0,0 +1,105 @@
+package com.codecool.virtualpub.logic;
+
+import com.codecool.virtualpub.data.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Pub {
+    private List<Drink> stock;
+    private List<Customer> customers;
+    private Bartender bartender;
+    private int profit;
+
+    public Pub(ArrayList<Drink> stock, ArrayList<Customer> customers, Bartender bartender) {
+        this.stock = new ArrayList<>(stock);
```

Comment:
Really nice data hiding, very good in terms of encapsulation. We should talk about the cases why we don't always do it with this approach!